doc/source/serve/monitoring.md

```diff
@@ -565,8 +566,15 @@ The following metrics are exposed by Ray Serve:
      - The number of requests processed by this DeploymentHandle.
    * - ``ray_serve_deployment_queued_queries`` [*]
      - * deployment
-       * route
+       * application
+       * handle
      - The number of queries for this deployment waiting to be assigned to a replica.
+   * - ``ray_serve_num_ongoing_requests_at_replicas`` [*]
+     - * deployment
+       * application
+       * handle
+       * replica
+     - The current number of requests to this deployment that's been assigned and sent to execute on a replica.
```

Comment:
Yeah that's a good point - again this may lead to inaccuracies if there are multiple handles per actor, but that's not expected if people use our public APIs so I think it's worth reducing the overhead. I'll drop the handle tag.

Reply:
I think the "bottleneck" that's making the cardinality of this metric high is the replica tag. Dropping the handle tag won't really do anything because we expect the handles to live on different workers (and metrics are tagged with worker id by default already).

Reply:
Nice catch! We synced offline. The plan is to remove the `replica` tag instead of the `handle` tag.

Reply:
Synced offline. This can cause pretty high cardinality for this metric, so we'll drop the replica tag, and each handle only emits the total number of requests sent to (all) replicas.